samples/cellular/modem_shell/overlay-nrf700x-wifi-scan-only.conf

```diff
@@ -78,5 +78,6 @@ CONFIG_NET_MGMT_EVENT_STACK_SIZE=1024
 # Heap allocation should be changed when CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT
 # and CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT (which should be the same value) are changed.
 # 10 APs seem to require 1644 bytes more heap.
```

Comment:
This may not be valid anymore, I'm wondering if we should just remove this line. @trantanen, what do you think?